src/packages/core/modal/common/property-settings/property-settings-modal.element.ts

```diff
@@ -153,7 +153,7 @@ export class UmbPropertySettingsModalElement extends UmbModalBaseElement<
 		this._customValidationOptions.forEach((option) => {
 			option.selected = option.value === regEx;
 		});
-		console.log(this._customValidationOptions);
+		//console.log(this._customValidationOptions);
```

Comment:
Just remove this line completely, no need for these console.logs(), I think its left over from some debugging.